WoWPro_Leveling/Classic/Horde/01_12_Shururu_Mulgore.lua

```diff
@@ -84,7 +84,7 @@ A Mazzranache|QID|766|M|46.99,57.07|N|From Maur Raincaller.|
 
 h Bloodhoof Village|QID|1656|M|46.70,61.00|N|At Innkeeper Kauth.|
 T A Task Unfinished|QID|1656|M|46.62,61.09|N|To Innkeeper Kauth.|
-A Dwarven Digging|QID|746|LVL|6|M|47.51,60.17|N|From Baine Bloodhoof.|
+A Dwarven Digging|QID|746|M|47.51,60.17|N|From Baine Bloodhoof.|
```

Comment:
FYI:  I found that this quest was not available to me due to level restrictions.
Perhaps the pickup till after the turnin for [Sharing the Land|]?